app/models/pod.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Pod < ApplicationRecord
+  ONLINE_DAYS = 14.days
```

Comment:
I think I would name this `ACTIVE_DAYS` or something else with "active".

Also, I think I would move the comment from above the `active?` method to here, to explain what this is used for, as this is the common place for where this is used now and also where the number "14" lives now. Or maybe also keep a small comment with "... was online recently" at the `active?` method? :thinking: